src/Webhooks/Webhook.php

```diff
@@ -38,13 +38,31 @@ public static function instance()
             return self::$instance;
         }
 
-        public static function createSynchWebhooks()
+        public static function getClient()
         {
+            $activate = Oauth2\Activate::instance();
+            $apiParams = $activate::$params;
+
+            $accessToken = $activate->getAuthorizeToken()['token'];
+            $appId = Settings::instance()->getSettingsOption('account', 'app_id');
+            $instance = Settings::instance()->getSettingsOption('account', 'instance');
+
+            $apiParams['accessToken'] = $accessToken;
+            $apiParams['clientId'] = $appId;
+            $apiParams['instance'] = $instance;
```

Comment:
can we replace this with the method created in Course.php `getApiParams`, also i belive this method should be in the helpers file or in a more generic place ?

Reply:
I think the best place for this method is to be in Oauth2\Activate so ill move it there.

Reply:
Those are 2 different methods one to get only the params and another to get a client, but I think both should be in `Oauth2\Activate`

Reply:
So as this will be lots of refactoring work I'm thinking of spinning another ticket for this: https://administrate.atlassian.net/browse/PLAT-12051